camerakit/src/main/java/com/flurgle/camerakit/CameraView.java

```diff
@@ -295,12 +295,12 @@ public void start() {
                 break;
         }
 
-        sWorkerHandler.post(new Runnable() {
+        sWorkerHandler.postDelayed(new Runnable() {
             @Override
             public void run() {
                 mCameraImpl.start();
             }
-        });
+        }, 100);
```

Comment:
Is this really necessary? What is that solving?

Reply:
The whole NPE exists because of some weird sync issue between camera and activity -- sometimes after resume camera thread can't get capture or preview size. So this small delay should sync two threads and give android internals some time to prepare while being transparent for users.